checkstyle-sonar-plugin/src/main/java/org/sonar/plugins/checkstyle/CheckstyleConfiguration.java

```diff
@@ -46,18 +46,23 @@
 import com.puppycrawl.tools.checkstyle.api.CheckstyleException;
 import com.puppycrawl.tools.checkstyle.api.Configuration;
 
-public class CheckstyleConfiguration implements BatchExtension {
+@ExtensionPoint
+@ScannerSide
+public class CheckstyleConfiguration {
     public static final String PROPERTY_GENERATE_XML = "sonar.checkstyle.generateXml";
 
     private static final Logger LOG = LoggerFactory.getLogger(CheckstyleConfiguration.class);
 
     private final CheckstyleProfileExporter confExporter;
     private final RulesProfile profile;
-    private final Settings conf;
+    private final org.sonar.api.config.Configuration conf;
```

Comment:
Why full name is used ?
I do not see a conflict, please use import.

Reply:
conflict is within already used `com.puppycrawl.tools.checkstyle.api.Configuration`.